packages/lesswrong/components/sunshineDashboard/ModeratorActions.tsx

```diff
@@ -402,38 +385,29 @@ export const ModeratorActions = ({classes, user, currentUser, refetch, comments,
     </LWTooltip>
   </div>
 
-  const moderatorActionLog = <div>
-    {user.moderatorActions
-      .filter(moderatorAction => moderatorAction.active)
-      .map(moderatorAction => {
-        let averageContentKarma: number | undefined;
-        if (moderatorAction.type === LOW_AVERAGE_KARMA_COMMENT_ALERT) {
-          const mostRecentComments = sortBy(comments ?? [], 'postedAt').reverse();
-          ({ averageContentKarma } = isLowAverageKarmaContent(mostRecentComments ?? [], 'comment'));
-        } else if (moderatorAction.type === LOW_AVERAGE_KARMA_POST_ALERT) {
-          const mostRecentPosts = sortBy(posts ?? [], 'postedAt').reverse();
-          ({ averageContentKarma } = isLowAverageKarmaContent(mostRecentPosts ?? [], 'post'));
-        }
-
-        const suffix = typeof averageContentKarma === 'number' ? ` (${averageContentKarma})` : '';
-        const endedAt = moderatorAction.endedAt ? `, ends on ${moment(moderatorAction.endedAt).format('YYYY-MM-DD').toString()}` : '';
-
-        return <div key={`${user._id}_${moderatorAction.type}`}>
-          {`${MODERATOR_ACTION_TYPES[moderatorAction.type]}${suffix}${endedAt}`}
-        </div>;
-      })
-    }
-  </div>
-
+  const [anchorEl, setAnchorEl] = useState<any>(null);
```

Comment:
Not 100% sure how this works but I have a vague sense that this ought to be a `useRef`?  I remember seeing something similar recently... might be confused, not sure.